src/Security/RememberLoginHash.php

```diff
@@ -190,14 +204,21 @@ public static function generate(Member $member)
     /**
      * Generates a new hash for this member but keeps the device ID intact
      *
+     * @deprecated 5.3.0 Token renewal will be removed in 6.0.0
```

Comment:
```suggestion
     * @deprecated 5.3.0 Will be removed without equivalent functionality
```
This is the standardised message we use for this kind of deprecation.